components/datatypes/mediacollectiondetails.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/mediaCollectionDetails",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Media collection details information",
+  "type": "object",
+  "meta:status": "experimental",
+  "meta:extensible": true,
+  "description": "Media collection details information.",
+  "definitions": {
+    "mediaCollectionDetails": {
+      "properties": {
+        "xdm:timeSeries": {
```

Comment:
What is the purpose of adding this here ? timestamp and eventType are pre-included in a schema when you use any experience event class based schemas.

Reply:
We created the mediaCollectionDetails datatype to be used inside an array for the downloaded fieldgroup. Each of the mediaCollectionDetails in this array represents an individual event and needs to have specified the eventType and timestamp of the individual event (e.g. https://github.com/adobe/xdm/blob/c146e10808d2a1141c53f994867cd1151d8fb90a/components/fieldgroups/experience-event/experienceevent-media-collection-downloaded.example.1.json). The downloaded content represents a full media collection session with all events starting from sessionStart, ping, play, etc. to sessionComplete.